static/app/views/settings/organizationMembers/inviteBanner.tsx

```diff
@@ -0,0 +1,188 @@
+import styled from '@emotion/styled';
+
+import {Button} from 'sentry/components/button';
+import Card from 'sentry/components/card';
+import ExternalLink from 'sentry/components/links/externalLink';
+import {Tooltip} from 'sentry/components/tooltip';
+import {IconCommit, IconGithub, IconInfo, IconMail} from 'sentry/icons';
+import {t, tct} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {MissingMember} from 'sentry/types';
+
+type Props = {
+  missingMembers: {integration: string; users: MissingMember[]};
+  onSendInvite: (email: string) => void;
+};
+
+export function InviteBanner({missingMembers, onSendInvite}: Props) {
+  // TODO(cathy): include snoozing, docs link
+
+  const cards = missingMembers?.users?.slice(0, 5).map(member => (
+    <MemberCard key={member.userId} data-test-id={`member-card-${member.userId}`}>
+      <MemberCardContent>
+        <MemberCardContentRow>
+          <IconGithub size="sm" />
+          <StyledExternalLink href={`http://github.com/${member.userId}`}>
```

Comment:
nit: Should we move this into a constant?

Reply:
i was thinking of having a dict with a lambda function to generate these

Reply:
i will probably change this in the next iteration of this

Reply:
Is this resolved? If not can you add a todo item so you don't miss in future PRs?

Reply:
sure